torch/jit/__init__.py

```diff
@@ -638,6 +638,15 @@ def make_module(mod, _module_class, executor_options):
         _module_class = TopLevelTracedModule
     return _module_class(mod, **executor_options)
 
+def wrap_check_inputs(check_inputs):
+    if check_inputs is None:
+        return None
+
+    canon_check_inputs = []
```

Comment:
I think this one liner is more idiomatic python:`[{'forward': c} for c in check_inputs}]`

Reply:
fixed! thanks!